tasks/task07/order_repository.py

```diff
@@ -0,0 +1,69 @@
+import uuid
+import time
+from typing import List
+
+
+class Good:
+    def __init__(self, id: uuid.UUID, name, price: int):
+        self.id = id
+        self.name = name
+        self.price = price
+
+
+class Order:
+    def __init__(self, order_id: uuid.UUID, order_date, client_id, goods: List[Good], price=0):
+        self.order_id = order_id
+        self.order_date = order_date
+        self.client_id = client_id
+        self.goods = goods
+        self.price = price
+
+    @property
+    def price(self):
+        return sum(g.price for g in self.goods)
+
+    @price.setter
+    def price(self, value):
+        self._price = value
+
+
+class OrderRepository:
+    orders = []
+
+    def add(self, order: Order):
+        self.orders.append(order)
+
+    def get(self, order_id: uuid.UUID) -> Order:
+        return [o for o in self.orders if o.order_id == order_id].pop(0)
+
+    def list(self, n_latest: int = None) -> List[Order]:
+        if n_latest is None:
+            return self.orders
+        else:
+            return self.orders[-n_latest:]
+
+    def delete(self, order_id):
+        order_to_be_deleted = [o for o in self.orders if o.order_id == order_id].pop(0)
```

Comment:
You have method `get` and you don't need found element manual for all method.
And you can use `self.orders.remove(self.get(order_id))`